packages/marshal/src/marshal-justin.js

```diff
@@ -31,16 +31,30 @@ const { stringify: quote } = JSON;
 const makeYesIndenter = () => {
   const strings = [];
   let level = 0;
-  const line = () => strings.push('\n', '  '.repeat(level));
+  let needSpace = false;
+  const line = () => {
+    needSpace = false;
+    return strings.push('\n', '  '.repeat(level));
+  };
   return harden({
     open: openBracket => {
-      assert(level >= 1);
```

Comment:
Wow.  How'd we miss this one?

Reply:
Classic one bid masking another bug. The call that was supposed to test this out was (and still is)
```js
const justinExpr = decodeToJustin(encoding, true);
```
because I thought, when I wrote it, that `shouldIndent` was the second parameter. Turns out that `cyclePolicy` was the second parameter, and the old code was still letting `shouldIndent` default to falsy. So the yes-indenter was not being used. Once I fixed the call, then this assertion started failing and I had a "Doh!" moment.

Reply:
Now that the `cyclePolicy` parameter has gone away, those calls are now correct.